README.md

```diff
@@ -38,44 +38,31 @@ Composable Router contracts consist of the following components:
 
 `forge coverage --rpc-url https://rpc.ankr.com/eth --report summary`
 
-### Deploy All
+### Deploy Contract(s)
 
-Fill out parameters in `script/deployParameters/<network>.json`
+Fill out parameters in `script/Deploy<network>.s.sol`
+
+- This script deploys all contracts whose `deployedAddress` equals `UNDEPLOYED`.
 
 ```console
 forge script --broadcast \
 --rpc-url <RPC-URL> \
 --private-key <PRIVATE-KEY> \
---sig 'run(string)' \
-script/DeployAll.s.sol:DeployAll \
-<pathToJSON>
+--sig 'run()' \
+script/Deploy<network>.s.sol:Deploy<network> \
```

Comment:
```suggestion
script/Deploy<network>.s.sol\
```
That also works. Which one do you prefer?

Reply:
I think I would prefer `script/Deploy<network>.s.sol:Deploy<network>` because this is straightforward that we call `Deploy<network>` contract. Though both of them work, I found another [tutorial](https://ethereum-blockchain-developer.com/2022-06-nft-truffle-hardhat-foundry/16-foundry-deployment/) and [Uniswap](https://github.com/Uniswap/universal-router/blob/main/README.md#to-deploy) also use the longer one (to avoid any unexpected error I guess).

Reply:
Sure.